x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_selectors.tsx

```diff
@@ -21,27 +26,36 @@ interface PolicyTemplateSelectorProps {
   disabled: boolean;
 }
 
+const getPolicyTemplateLabel = (policyTemplate: CloudSecurityPolicyTemplate) => {
+  if (policyTemplate === VULN_MGMT_POLICY_TEMPLATE) {
+    return CNVM_POLICY_TEMPLATE.toUpperCase();
+  }
+  return policyTemplate.toUpperCase();
```

Comment:
we should use localized strings whenever we display text to our users

Lets map these codes to their relevant text instead of using technical text to display to the user